### DISCUSSION TITLE
Monorepo: how to have no secondary clone?

### DISCUSSION THREAD

**Discussion by namrog84** [renovatebot/renovate#12616] [Request Help] [answered]
We have 1 monorepo, so our renovate.json and config.js all live in the same repo.  Been dealing with some auth related issues and 1 semi problematic thing is we already are cloning the whole repo to run renovate, but then it clones the exact same repo again.

{"name":"renovate","hostname":"78f154e0c000001","pid":126246,"level":20,"logContext":"fL-HFVT4u","repository":"myorg/myproj","msg":"Performing blobless clone","time":"2021-11-12T00:34:37.095Z","v":0}
{"name":"renovate","hostname":"78f154e0c000001","pid":126246,"level":20,"logContext":"fL-HFVT4u","repository":"myorg/myproj","durationMs":189887,"msg":"git clone completed","time":"2021-11-12T00:37:46.982Z","v":0}

Is there a way to just have it run in the same repo that its already running in?

I suppose I can try setting the baseDir or cacheDir to the place it's already cloned and hope it doesn't reclone some modified files?

**Comment by rarkins** [comment, marked as answer]
No, that's not possible. Why not just put your Renovate logic in a separate repo, e.g. containing the config.js?

**Reply by namrog84** [reply]
We can do that.  I am just so used to working in our mega monorepo, but since the seperate repo and config will almost never need to be changed, I suppose the renovate.json living in the main repo still makes sense to me and will get proper visibility.
